### PR TITLE
PDCT-370 Hide page label and target date if empty or null.

### PR DESCRIPTION
**Pull request by katybaulch**

### CODE REVIEW

File: src/components/Targets.tsx [hunk 1]
```diff
@@ -19,13 +19,19 @@ export const Targets = ({ targets = [], showFamilyInfo = false }: TTargets) => {
...
+  };
+
   return (
     <ul className="" data-cy="targets">
       {targets.map((target) => (
         <li className="mt-6 flex flex-col gap-2 border-b pb-2 last:border-0" key={target.ID}>
           <span dangerouslySetInnerHTML={{ __html: target.Description }} className="" />
           <span className="text-gray-700 text-sm">
-            {`${target.Sector}${target.Scopes !== "" ? `: ${target.Scopes}` : ""}`} &middot; Target year: {target.Year}
+            {/* {`${target.Sector}${target.Scopes !== "" ? `: ${target.Scopes}` : ""}`} &middot; Target year: {target.Year} */}
```
Comment (PatFawbertMills) [inline src/components/Targets.tsx]: Hey look there's a comment

File: src/components/drawer/FamilyMatchesDrawer.tsx [hunk 2]
```diff
@@ -27,6 +28,12 @@ export const FamilyMatchesDrawer = ({ family }: TProps) => {
     router.push({ pathname: `/documents/${document.document_slug}`, query: queryObj });
   };
 
+  const showPageNumbers = (documentContentType: TDocumentContentType) => {
+    if (!documentContentType) return null;
```
Comment (diversemix) [inline src/components/drawer/FamilyMatchesDrawer.tsx:32]: I'm intrigued, why are we returning null?

**Review by PatFawbertMills — APPROVED** [review]
LGTM

**Review by diversemix — APPROVED** [review]
lgtm :+1:

File: src/components/drawer/FamilyMatchesDrawer.tsx (hunk 2 above)
Comment (katybaulch) [reply src/components/drawer/FamilyMatchesDrawer.tsx]: Changed to explicitly return false instead of falsey null.

**Pull request merged**